e2e/impl/tokens.go

```diff
@@ -32,6 +32,46 @@ func (ctx *ValidationContext) sendTokensAndWait(from, to accounts.Account, token
 }
 
 func (ctx *ValidationContext) sendTokens(from, to accounts.Account, tokens int64) error {
+	weis := toWei(tokens)
+	hexWeis := hexutil.Big(*weis)
+	args := knode.TransferArgs{
+			From:  from.Address,
+			To:    &to.Address,
+			Value: &hexWeis,
+	}
+
+	res := &cluster.ExecResponse{}
+	return ctx.execCommand(transferTokens(args), res)
```

Comment:
We could wait for the tokens to be transfered too.

Reply:
This function is without waiting and it's used in `sendTokensAndWait` that wait for the tx committed.